pkg/controllers/common/utils.go

```diff
@@ -56,7 +56,12 @@ func AllocateSubnetFromSubnetSet(subnetSet *v1alpha1.SubnetSet, vpcService servi
 		log.Error(err, "Failed to allocate Subnet")
 		return "", err
 	}
-	return subnetService.CreateOrUpdateSubnet(subnetSet, vpcInfoList[0], tags)
+	subnetPath, err := subnetService.CreateOrUpdateSubnet(subnetSet, vpcInfoList[0], tags, false)
+	if err != nil && strings.Contains(err.Error(), "property subnet_dhcp_config is unrecognized") {
```

Comment:
I saw the same string `property subnet_dhcp_config is unrecognized` many times in this PR. Could we define a constant here to avoid hard coding and improve code maintainability? This way, if the error message changes in the future, it can be updated in one place. For example:
```
const ErrUnrecognizedSubnetDHCPConfig = "property subnet_dhcp_config is unrecognized"
```

Reply:
Updated, thanks!